<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nExecutionEditor</class>
 <widget class="QWidget" name="nExecutionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Execution editor</string>
  </property>
  <property name="toolTip">
   <string>Execution editor</string>
  </property>
  <property name="statusTip">
   <string>Execution editor</string>
  </property>
  <property name="whatsThis">
   <string>Execution editor</string>
  </property>
  <widget class="QToolButton" name="ProgramIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Program icon</string>
   </property>
   <property name="statusTip">
    <string>Program icon</string>
   </property>
   <property name="whatsThis">
    <string>Program icon</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>240</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Name</string>
   </property>
   <property name="statusTip">
    <string>Name</string>
   </property>
   <property name="whatsThis">
    <string>Name</string>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="registerEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>240</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Registration</string>
   </property>
   <property name="statusTip">
    <string>Registration</string>
   </property>
   <property name="whatsThis">
    <string>Registration</string>
   </property>
   <property name="placeholderText">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Executable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>330</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Executable</string>
   </property>
   <property name="statusTip">
    <string>Executable</string>
   </property>
   <property name="whatsThis">
    <string>Executable</string>
   </property>
   <property name="placeholderText">
    <string>Executable</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Paraments">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>330</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Paraments</string>
   </property>
   <property name="statusTip">
    <string>Paraments</string>
   </property>
   <property name="whatsThis">
    <string>Paraments</string>
   </property>
   <property name="placeholderText">
    <string>Paraments</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Environment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>330</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Environment</string>
   </property>
   <property name="statusTip">
    <string>Environment</string>
   </property>
   <property name="whatsThis">
    <string>Environment</string>
   </property>
   <property name="placeholderText">
    <string>Environment</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="identifierBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Identifier</string>
   </property>
   <property name="statusTip">
    <string>Identifier</string>
   </property>
   <property name="whatsThis">
    <string>Identifier</string>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="parallelBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Parallel execution</string>
   </property>
   <property name="statusTip">
    <string>Parallel execution</string>
   </property>
   <property name="whatsThis">
    <string>Parallel execution</string>
   </property>
   <property name="text">
    <string>Parallel execution</string>
   </property>
  </widget>
  <widget class="QToolButton" name="browseExecute">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Images/Branon.qrc">
     <normaloff>:/images/NewProject.png</normaloff>:/images/NewProject.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="browseEnviron">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Images/Branon.qrc">
     <normaloff>:/images/version.png</normaloff>:/images/version.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../Images/Branon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ProgramIcon</sender>
   <signal>clicked()</signal>
   <receiver>nExecutionEditor</receiver>
   <slot>ProgramIcon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseExecute</sender>
   <signal>clicked()</signal>
   <receiver>nExecutionEditor</receiver>
   <slot>BrowseExecution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseEnviron</sender>
   <signal>clicked()</signal>
   <receiver>nExecutionEditor</receiver>
   <slot>BrowseEnvironment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
